go.mod

```diff
@@ -111,7 +112,7 @@ require (
 	github.com/onsi/gomega v1.5.0
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/image-spec v1.0.1 // indirect
-	github.com/opencontainers/runc v0.0.0-20181113202123-f000fe11ece1
+	github.com/opencontainers/runc v1.0.0-rc2.0.20190611121236-6cc515888830
```

Comment:
how was this revision selected? can we pin to a tagged release?

Reply:
It is the first commit including the required fix in runc. https://github.com/opencontainers/runc/pull/2065. They have no release containing the new fix.

Reply:
I can rebase to the newest commit f4982d86f7fde0b6f953cc62ccc4022c519a10a9, but there has been no code changes since `6cc515888830`